tests/integration_tests/functional/test_snapshot_basic.py

```diff
@@ -120,17 +121,19 @@ def test_5_snapshots(
         microvm = microvm_factory.build()
         microvm.spawn()
         microvm.restore_from_snapshot(snapshot, resume=True)
+        vm.ssh.run("pkill -SIGCONT socat")
```

Comment:
Do we really need this SIGSTOP/SIGCONT pair?

Reply:
https://github.com/firecracker-microvm/firecracker/pull/4278#discussion_r1407868707